RedBookPlayer.GUI/App.xaml.cs

```diff
@@ -24,17 +30,19 @@ public override void Initialize()
                 Console.WriteLine(((Exception)f.ExceptionObject).ToString());
             };
 
+            Settings = SettingsViewModel.Load("settings.json");
```

Comment:
what happens if there's no json?
or if it is wrong?

Reply:
Then it uses sane defaults for anything it can't load. It creates the JSON the first time that settings change.